File: packages/insomnia-httpsnippet/test/fixtures/output/java/okhttp/text-plain.java
OkHttpClient client = new OkHttpClient();

MediaType mediaType = MediaType.parse("text/plain");
RequestBody body = RequestBody.create(mediaType, "Hello World");
Request request = new Request.Builder()
  .url("http://mockbin.com/har")
  .post(body)
  .addHeader("content-type", "text/plain")
  .build();

Response response = client.newCall(request).execute();

File: packages/insomnia-httpsnippet/test/fixtures/output/node/request/text-plain.js
var request = require('request');

var options = {
  method: 'POST',
  url: 'http://mockbin.com/har',
  headers: { 'content-type': 'text/plain' },
  body: 'Hello World'
};

request(options, function(error, response, body) {
  if (error) throw new Error(error);

  console.log(body);
});
